api/camunda/v2/authorizations.yaml

```diff
@@ -0,0 +1,367 @@
+paths:
+  /authorizations:
+    post:
+      tags:
+        - Authorization
+      operationId: createAuthorization
+      summary: Create authorization
+      description: |-
+
+
+        [[CONSISTENCY:STRONG]]
+
+        Create the authorization.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/AuthorizationCreateResult"
+        required: true
+      responses:
+        "201":
+          description: The authorization was created successfully.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/AuthorizationResult"
+        "400":
+          $ref: common-responses.yaml#/components/responses/InvalidData
```

Comment:
Ok now it should show everything correctly, but I still don't understand why the quotes are removed sometimes 👀  @tmetzke 

I'll take another look tomorrow